strip-coverletter-docker.sh

```diff
@@ -1,19 +1,40 @@
 #!/bin/bash
 set -eu
-set -xv
+
 in=$(realpath $1)
 bname_in=${in##*/} # /foo/bar/baz.pdf = baz.pdf
 
 out=$2
+out_dir=$(dirname $out)
```

Comment:
Is ``out_dir`` used below?

Reply:
ah, I was in the wrong script. no, it doesn't look like it's being used anymore. good catch